### PR TITLE
chore: GTM migration plan part 2

### PR DESCRIPTION
**Pull request by Chef-Yogi**
add token highlight event tracking

### CODE REVIEW

File: apps/web/src/views/Swap/components/HotTokenList/SwapTokenTable.tsx [hunk 1]
```diff
@@ -211,6 +212,12 @@ const DataRow: React.FC<
               e.preventDefault()
               const currency = new Token(chainId, address, tokenData.decimals, tokenData.symbol)
               handleOutputSelect(currency)
+              customGTMEvent?.push({
+                event: 'eventTracking',
+                category: 'TokenHighlight',
```
Comment (chef-jojo) [inline apps/web/src/views/Swap/components/HotTokenList/SwapTokenTable.tsx:217]: Let's make enum for all possibilities

Comment (Chef-Yogi) [reply apps/web/src/views/Swap/components/HotTokenList/SwapTokenTable.tsx]: @chef-jojo Done

File: apps/web/src/views/Swap/components/HotTokenList/SwapTokenTable.tsx [hunk 2]
```diff
@@ -211,6 +212,12 @@ const DataRow: React.FC<
               e.preventDefault()
               const currency = new Token(chainId, address, tokenData.decimals, tokenData.symbol)
               handleOutputSelect(currency)
+              customGTMEvent?.push({
```
Comment (Chef-Yogi) [inline apps/web/src/views/Swap/components/HotTokenList/SwapTokenTable.tsx:215]: add enum

File: apps/web/src/views/Swap/components/HotTokenList/SwapTokenTable.tsx [hunk 3]
```diff
@@ -211,6 +212,12 @@ const DataRow: React.FC<
...
+                event: GTMEvent.EventTracking,
+                category: GTMCategory.TokenHighlight,
+                action: GTMAction.ClickTradeButton,
+                label: tokenData.symbol,
+              })
```
Comment (Chef-Yogi) [inline apps/web/src/views/Swap/components/HotTokenList/SwapTokenTable.tsx:220]: enum added

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**